test/com/facebook/buck/apple/testdata/app_bundle_with_cxx_library/dep/BUCK

```diff
@@ -0,0 +1,9 @@
+cxx_library(
```

Comment:
You need to add `.fixture` extension to this file

Reply:
Oops, fixed now